src/plugins/legacy/itkProcessRegistrationOptimus/itkProcessRegistrationOptimusToolBox.cpp

```diff
@@ -0,0 +1,257 @@
+/*=========================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2013. All rights reserved.
+ See LICENSE.txt for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+=========================================================================*/
+
+#include "itkProcessRegistrationOptimus.h"
+#include "itkProcessRegistrationOptimusToolBox.h"
+
+#include <QtGui>
+
+#include <medAbstractDataFactory.h>
+#include <medAbstractData.h>
+#include <medAbstractRegistrationProcess.h>
+
+#include <medRunnableProcess.h>
+#include <medJobManagerL.h>
+#include <medAbstractImageData.h>
+
+#include <medToolBoxFactory.h>
+#include <medRegistrationSelectorToolBox.h>
+#include <medProgressionStack.h>
+#include <medPluginManager.h>
+#include <rpiCommonTools.hxx>
+
+class itkProcessRegistrationOptimusToolBoxPrivate
+{
+public:
+    QSpinBox * iterationsBox;
+    QSpinBox * binsBox;
+    QSpinBox * samplesBox;
+    QSpinBox * interpolationsBox;
+    QDoubleSpinBox * rhoStartBox;
+    QDoubleSpinBox * rhoEndBox;
+    QDoubleSpinBox * scalingCoeffBox;
+    QComboBox * interpolatorTypeBox;
+    medProgressionStack * progression_stack;
```

Comment:
the actual toolboxes use a common progression stack, you can remove this line

Reply:
ok